### ISSUE TITLE
Notifications on self-hosted version

### ISSUE THREAD

**Issue by JZafarr** [JhumanJ/OpnForm#389]
Hi there, 

How can i receive notifications on the self-hosted version. I have put in the SMTP details in the laravel.env and have subscribed to the "Email Notifications" integration but no notifications are received when the form is submitted.

Is this even possible on the self-hosted version?

Thanks

**Comment by JZafarr** [comment]
I even tried to put in a webhook URL but not receiving any notifications

**Comment by JhumanJ** [comment]
Is the queue properly running? Any logs to share? Does the forget password mail work?

**Comment by JZafarr** [comment]
I don't know how to check the queues, the container is up and running when running docker ps -a command.
The logs are here when requesting a password reset, I receive no emails when requesting for a password reset.

Docker logs after requesting a password reset:
https://gist.github.com/JZafarr/420eca7ba4d7b6d132d35ef333df743d

Looks like the email is getting generated but not getting sent to the email for some reason. I have redacted the email address and the links. 

I have tried 2 different SMTP settings, they appear to be working on another service. Here how the settings look like in the laravel.env file:
![Screenshot 2024-04-30 200230](https://github.com/JhumanJ/OpnForm/assets/62207359/d086fa2c-21ca-4325-8dcb-7a5d0aa7a4e8)
